lib/winrm-fs/core/tmp_zip.rb

```diff
@@ -111,7 +111,7 @@ def clean_dirname(dir)
         #   directory, excluding directories
         # @api private
         def entries
-          Pathname.glob(dir.join('**/*')).delete_if(&:directory?).sort
+          Pathname.glob(dir.join('**/.*')).push(*Pathname.glob(dir.join('**/*'))).delete_if(&:directory?).sort
         end
```

Comment:
hmm. It is very surprising to me that `**/*` would not include `**/.*`

Reply:
It doesn't, not even a linux shell glob includes dot files, we must be explicit.